src/main/java/ymfc/parser/Parser.java

```diff
@@ -88,6 +91,11 @@ public static Command parseCommand(String commandString, RecipeList recipes)
                 throw new EmptyListException("There is nothing to find!");
             }
             return getFindCommand(args);
+        case "findI":
+            if (numIngredients <= 0) {
+                throw new EmptyListException("There are no ingredients to find!");
```

Comment:
In the future we might consider changing the find recipe command to "findR" too then, to maintain consistency with the "listI" and "listR" commands